ext/afform/admin/ang/afGuiEditor/afGuiEditor.component.js

```diff
@@ -260,6 +267,12 @@
         return options;
       }
 
+      this.getLink = function() {
+        if (editor.afform.server_route) {
+          return CRM.url(editor.afform.server_route, null, editor.afform.is_public ? 'front' : 'back');
```

Comment:
@colemanw I'm just wondering do you think by making this change we need to make change to the help text for this. At the moment it just says `Style page to match front-end website` and I'm not sure if that is equal to "This url is accessible from the front end site like other public forms" ?

Reply:
Yea, TBH I don't think `is_public` has anything to do with styling (maybe in future it could) as the only effect it seems to have is this:
https://github.com/civicrm/civicrm-core/blob/5c89828ccfa4f48906ad041b68cebccbb736849d/ext/afform/core/afform.php#L397

I don't think that has any effect in Drupal, but alters the URL path in WP & J!